blueocean-dashboard/src/main/less/run-details-artifact.less

```diff
@@ -0,0 +1,8 @@
+.downloadAllArtifactsButton {
+    text-align: center
+}
```

Comment:
Just wanted to double check this is needed - I had thought that `.btn-secondary` would apply `text-align: center` to the `a` tag's text.